src/services/PushService.ts

```diff
@@ -43,7 +43,7 @@ export const registerLocalNotification = async (title: string, message: string,
 
     notification
       .android.setChannelId('LocalPush')
-      .android.setSmallIcon('ic_launcher')
+      .android.setSmallIcon('ic_beenhere_icon_notifi')
```

Comment:
no reason not to be consistent and call this `ic_been_here_icon_notifi`.  
Also, _[Notif](https://en.wiktionary.org/wiki/notif)_ is a more known shortening of notification.